hubstorage/serialization.py

```diff
@@ -1,4 +1,5 @@
 from json import dumps, loads
+from msgpack import Unpacker
```

Comment:
"extra dependency" is what Alex proposes. The usual problem with this approach is that same version can behave differently depending on the environment, so any bug or difference introduced by new encoder will be harder to spot.